build.gradle

```diff
@@ -23,7 +24,7 @@ ext {
     licenseUrl = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
     localGHPagesRepoDir = '../grain.gh-pages'
     localGHPagesRepoCanonicalPath = new File(localGHPagesRepoDir).canonicalPath
-    javaCompatibilityVersion = 1.7
+    javaCompatibilityVersion = 1.8
```

Comment:
Yes, the point is for current code we don't use features of Java 8, so its better to support both as it doesn't "cost" as anything.